package.json

```diff
@@ -7,10 +7,14 @@
     "@testing-library/react": "^12.1.2",
```

Comment:
- [ ] I used this place to give a suggestion about your `public` folder. In order to make your codebase cleaner you can delete all boilerplate text, images, etc. from the `public `folder.

Reply:
Thank you for the review. I have implemented your suggestion